### ISSUE TITLE
NPC Antonio Perelli

### ISSUE THREAD

**Issue by Orcozzio** [scotty0100/wowascension-issues#271]
Name: Gnocca
Account Name (for vote points rewards): Orcozzio
Map & Area: Elwynn Forest - along the road from Redridge to Westfall, near the bridge where is Guard Thomas

How to replicate the bug: try to walk the path in search of the npc and when you find him try to talk with him also take a look on his walking path

What happens: sometimes you see him standing still on the road, but you cannot talk with him... you see a red message "that merchant doesn't like you"... after a few seconds he disappear, but you can see him again few yards away... now you can go to talk with him

What should happen: You should always be able to stop any sales(we)man npc in order to buy/sell/browse his/her stuff

Located on WoWHead Map (if applicable):
Source (proof, if applicable): 

Extra Information: this issue looks very similar to what happens, for example, with some guards npc inside Stormwind... sometimes they look walking like levitating in the air shacking arms

**Comment by Skylark14** [comment]
This is a known bug with some salesmen.  You only have to click again on them to be able to buy from them.

**Comment by Orcozzio** [comment]
yes... that works in most cases, but not with Perelli... when you see him standing still in certain positions along the road you cannot interact with him... tryed click-spamming on him and always I see a red message saying "that merchant doesn't like you"... I made a screenshot but dunno how to upload it here in github

**Comment by Skylark14** [comment]
He does work, I vendored some stuff to him on my paladin the other day.  If he doesn't 'like you', move away from him some distance and then go back, this usually allows you to buy from him.

**Comment by Faraday1** [comment]
He works, sometimes when he resets his pathing you get the message he doesn't like you. Just have to try him again.

**Comment by kamicolo** [comment]
This isn't fixed so opening again
